tasks/pipeline.py

```diff
@@ -629,9 +627,14 @@ def edit_schedule(_, schedule_id, description=None, ref=None, cron=None, cron_ti
     Edit an existing pipeline schedule on the repository.
     """
 
-    gitlab = _init_pipeline_schedule_task()
-    result = gitlab.edit_pipeline_schedule(schedule_id, description, ref, cron, cron_timezone)
-    pprint.pprint(result)
+    repo = get_gitlab_repo(token=get_gitlab_bot_token())
+
+    data = {'description': description, 'ref': ref, 'cron': cron, 'cron_timezone': cron_timezone}
+    data = {key: value for (key, value) in data.items() if value is not None}
+
+    sched = repo.pipelineschedules.update(schedule_id, data)
+
+    pprint.pprint(sched)
```

Comment:
HA 😄 
But I'm not sure about what it would output (if it's similar to what it was).